### DISCUSSION TITLE
[PROPOSAL] Fishing (SFL) - Proposal Combined Part 1 and 2 by Jc Eii

### DISCUSSION THREAD

**Discussion by JcEii** [sunflower-land/sunflower-land#1129] [Ideas]

**Comment by JcEii** [comment]


**Reply by esp123** [reply]
looking forward master to your next big proposals

**Comment by d4nnY132** [comment]
You created a very big work,love it!

**Reply by JcEii** [reply]
Thank you sir!

**Comment by phix06** [comment]
Juste le type a pas de vie pour autant d'investissement personnel ;;;; je kiffe tt ce tu as crée cest énorme ton projet est génial BRAVOOOO

**Reply by JcEii** [reply]
Merci Monsieur!

**Comment by On1el** [comment]
You forgot to add one very rare fis
![SHARK](https://user-images.githubusercontent.com/104461938/178359201-813739ee-628f-4210-b4bd-f48c819238e1.png)
h

**Comment by JcEii** [comment]


**Reply by JcEii** [reply]
This may be the last, thank you everyoneeeeee <3

**Comment by SuperSynapse** [comment]
I'm still amazed how radio silent this post has been.

Incredible content and I'm excited for the hope that one day it'll make it into SFL.

BTW happy Anniversary on your first post in 3 days! 🎂 May 14th
